scripts/generator.ts

```diff
@@ -99,9 +73,11 @@ export class Generator {
    */
   private getAPIs(items) {
```

Comment:
While you're editing this code, can you add a type for `items`?

Reply:
Method wasn't even being used lol, deleted